graylog2-web-interface/src/components/indexers/IndexerFailuresComponent.jsx

```diff
@@ -50,21 +50,24 @@ const IndexerFailuresComponent = React.createClass({
     return 'ambulance';
   },
   render() {
+    let content;
     if (this.state.total === undefined) {
-      return <Spinner />;
+      content = <Spinner />;
+    } else {
+      content = this._formatFailuresSummary();
     }
+
     return (
       <Row className="content">
         <Col md={12}>
-          <h2><i className="fa fa-truck" /> Indexer Failures</h2>
```

Comment:
We decided long ago to remove all icons from headers (basically in the UI changes we started in 1.1), as in the end they are mostly random and don't provide any value. The system overview page is the only place that still got some of them, and they weren't even used consistently, so some sections had them, and some not.

Now that I was adding a new section into the page, I decided it was time to get rid of them.